### DISCUSSION TITLE
Missing shipping_email in Sellbrite-API /orders response

### DISCUSSION THREAD

**Discussion by cyhsujohn** [Sellbrite/Sellbrite-API#9] [Q&A] [answered]
Hi,

We found the response of Sellbrite-API /orders endpoints(including /orders/:sb_order_seq (GET) and /warehouses/fulfillments/:uuid (GET)) will only return `null` as `shipping_email`.

Based on [the documentation](https://developer.sellbrite.com/endpoints/orders#:~:text=String-,shipping_email,-The%20email%20of), we should have received the shipping_email. We tried to check for multiple orders belonging to different stores and the results are all the same(shipping_email would be `null`).

Are there any issues with the API?

**Comment by rengler-godaddy** [comment]
Hey John - Thanks for reaching out. 

Can you send me an example of the response you’re receiving, along with the email address for the account you're pulling the request from?

**Comment by cyhsujohn** [comment]
Hi Jengler,

We're using the account "tommaso@easyship.com" to pull the request. 
The response attached below is for both endpoints:
```
GET https://api.sellbrite.com/v1/warehouses/fulfillments/e7b372ae-cd78-47d9-951c-aeb2d734ab91
GET https://api.sellbrite.com/v1/orders/1011
```

Response:
```
{"order_ref"=>"1047",
 "display_ref"=>"#1047",
 "sb_order_seq"=>1011,
 "shipping_contact_name"=>"John Hsu",
 "shipping_email"=>nil,
 "shipping_company_name"=>nil,
 "shipping_address_1"=>"55 Prospect Street",
 "shipping_address_2"=>"Unit 401,",
 "shipping_address_3"=>nil,
 "shipping_postal_code"=>"11201",
 "shipping_state_region"=>"NEW YORK",
 "shipping_city"=>"BROOKLYN",
 "shipping_country_code"=>"US",
 "shipping_phone_number"=>nil,
 "subtotal"=>5.0,
 "discount"=>0.0,
 "shipping_cost"=>16.44,
 "total"=>21.44,
 "tax"=>0.0,
 "sb_payment_status"=>"all",
 "shipment_status"=>"none",
 "ordered_at"=>"2023-05-18T12:58:41Z",
 "sb_status"=>"open",
 "cancelled_at"=>nil,
 "shipped_at"=>nil,
 "billing_address_1"=>"55 Prospect Street",
 "billing_address_2"=>"Unit 401,",
 "billing_address_3"=>nil,
 "billing_postal_code"=>"11201",
 "billing_state_region"=>"NEW YORK",
 "billing_city"=>"BROOKLYN",
 "billing_country"=>"UNITED STATES",
 "billing_country_code"=>"US",
 "billing_phone_number"=>nil,
 "billing_contact_name"=>"John Hsu",
 "billing_email"=>"john.hsu@easyship.com",
 "billing_company"=>nil,
 "channel_uuid"=>"a85a96e8-2125-466e-a736-170ae848939e",
 "channel_name"=>"shopify john-staging-test-2",
 "channel_type_display_name"=>"Shopify",
 "requested_shipping_service"=>"OSM - Domestic Parcel ORD CF",
 "requested_shipping_provider"=>nil,
 "customer_notes"=>nil,
 "merchant_notes"=>nil,
 "ship_by_date"=>nil,
 "deliver_by_date"=>nil,
 "requested_shipping_special"=>nil,
 "handling_cost"=>0.0,
 "paid_at"=>"2023-05-18T13:02:42Z",
 "items"=>
  [{"order_item_ref"=>"12459384668221",
    "quantity"=>1,
    "title"=>"1Test Item - South AUS",
    "sku"=>"1test-item-SAUS",
    "unit_price"=>5.0,
    "total"=>5.0,
    "quantity_fulfilled"=>0,
    "inventory_sku"=>"1test-item-SAUS",
    "tax"=>0.0,
    "cancelled_at"=>nil,
    "status"=>nil,
    "warehouses"=>[{"uuid"=>"e7b372ae-cd78-47d9-951c-aeb2d734ab91", "name"=>"WeWork"}]}]}
```
<img width="1479" alt="image" src="https://github.com/Sellbrite/Sellbrite-API/assets/65455378/64e0f12d-4aed-4bf0-a6ce-aecbfb490dc7">

**Comment by rengler-godaddy** [comment, marked as answer]
Hey John - Thanks for sending that request over. 

Any information that falls under the 'Sold To' section on the order will be under the billing data. So in this case the email is attached to the "billing_email" parameter. Some sales channels do allow a separate email to be sent for the shipping address, but usually, it's sent over under the billing/sold to section of the order. 

I hope this information helps! Let us know if you need anything else.

**Reply by cyhsujohn** [reply]
Thank you for getting back to me. We'll update our implementation about this.

**Comment by rengler-godaddy** [comment]
My pleasure!
